src/browser/commands.js

```diff
@@ -5,6 +5,8 @@ export { setViewport } from './reset.js';
 function getElementPosition(elem) {
 	const { x, y, width, height } = elem.getBoundingClientRect();
 	return {
+		left: Math.floor(x + window.scrollX),
+		top: Math.floor(y + window.scrollY),
```

Comment:
The only thing that might be an issue here is right-to-left, where the left/right would need to flip. Would it make any sense for the offsets to be from centre instead of top-left? 🤔

Reply:
Very true... and yeah the existing stuff clicking in the centre is already susceptible to this like you said. Let's ignore this for now -- if it comes up we can address it (or not) then.